boards/arm/pan1783_pan1783a_evb/pan1783a_evb_cpunet.dts

```diff
@@ -0,0 +1,215 @@
+			zephyr,code = <INPUT_KEY_2>;
+		};
+		evb_sw4: evb_sw_4 {
+			gpios = <&gpio0 9 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
+			label = "SW4 on EVB";
+			zephyr,code = <INPUT_KEY_3>;
+		};
+	};
+
+	mikrobus_header: mikrobus-connector {
+		compatible = "mikro-bus";
+		#gpio-cells = <2>;
+		gpio-map-mask = <0xffffffff 0xffffffc0>;
+		gpio-map-pass-thru = <0 0x3f>;
+		gpio-map =	<0 0 &gpio0 4 0>,	/* AN  */
+				/* Not a GPIO*/		/* RST */
+				<2 0 &gpio1 12 0>,	/* CS   */
+				<3 0 &gpio1 15 0>,	/* SCK  */
+				<4 0 &gpio1 14 0>,	/* MISO */
+				<5 0 &gpio1 13 0>,	/* MOSI */
+							/* +3.3V */
+							/* GND */
+				<6 0 &gpio1 7 0>,	/* PWM  */
+				<7 0 &gpio1 4 0>,	/* INT  */
+				<8 0 &gpio1 0 0>,	/* RX   */
+				<9 0 &gpio1 1 0>,	/* TX   */
+				<10 0 &gpio1 3 0>,	/* SCL  */
+				<11 0 &gpio1 2 0>;	/* SDA  */
+							/* +5V */
+							/* GND */
+	};
+
+	arduino_header: connector {
+		compatible = "arduino-header-r3";
+		#gpio-cells = <2>;
+		gpio-map-mask = <0xffffffff 0xffffffc0>;
+		gpio-map-pass-thru = <0 0x3f>;
+		gpio-map = <0 0 &gpio0 4 0>,	/* A0 */
+			   <1 0 &gpio0 5 0>,	/* A1 */
+			   <2 0 &gpio0 6 0>,	/* A2 */
+			   <3 0 &gpio0 7 0>,	/* A3 */
+			   <4 0 &gpio0 25 0>,	/* A4 */
+			   <5 0 &gpio0 26 0>,	/* A5 */
+			   <6 0 &gpio1 0 0>,	/* D0 */
+			   <7 0 &gpio1 1 0>,	/* D1 */
+			   <8 0 &gpio1 4 0>,	/* D2 */
+			   <9 0 &gpio1 5 0>,	/* D3 */
+			   <10 0 &gpio1 6 0>,	/* D4 */
+			   <11 0 &gpio1 7 0>,	/* D5 */
+			   <12 0 &gpio1 8 0>,	/* D6 */
+			   <13 0 &gpio1 9 0>,	/* D7 */
+			   <14 0 &gpio1 10 0>,	/* D8 */
+			   <15 0 &gpio1 11 0>,	/* D9 */
+			   <16 0 &gpio1 12 0>,	/* D10 */
+			   <17 0 &gpio1 13 0>,	/* D11 */
+			   <18 0 &gpio1 14 0>,	/* D12 */
+			   <19 0 &gpio1 15 0>,	/* D13 */
+			   <20 0 &gpio1 2 0>,	/* D14 */
+			   <21 0 &gpio1 3 0>;	/* D15 */
+	};
+
+	/* These aliases are provided for compatibility with samples */
+	aliases {
+		led0 = &evb_led1;
+		led1 = &evb_led2;
+		led2 = &evb_led3;
+		led3 = &evb_led4;
+		sw0 = &evb_sw1;
+		sw1 = &evb_sw2;
+		sw2 = &evb_sw3;
+		sw3 = &evb_sw4;
+		bootloader-led0 = &evb_led1;
+		mcuboot-button0 = &evb_sw1;
+		mcuboot-led0 = &evb_led1;
+		watchdog0 = &wdt0;
+	};
+};
+
+&gpiote {
+	status = "okay";
+};
+
+&gpio0 {
+	status = "okay";
+};
+
+&gpio1 {
+	status = "okay";
+};
+
+&uart0 {
+	status = "disabled";
+	current-speed = <115200>;
+	pinctrl-0 = <&uart0_default>;
+	pinctrl-1 = <&uart0_sleep>;
+	pinctrl-names = "default", "sleep";
+};
+
+arduino_serial: &uart0{};
+
+arduino_i2c: &i2c0 {
+	compatible = "nordic,nrf-twim";
+	/* Cannot be used together with uart0. */
+	/* status = "okay"; */
+	pinctrl-0 = <&i2c0_default>;
+	pinctrl-1 = <&i2c0_sleep>;
+	pinctrl-names = "default", "sleep";
+};
+
+arduino_spi: &spi0 {
+	compatible = "nordic,nrf-spim";
+	/* Cannot be used together with uart0. */
+	/* status = "okay"; */
+	cs-gpios = <&arduino_header 16 GPIO_ACTIVE_LOW>; /* D10 */
+	pinctrl-0 = <&spi0_default>;
+	pinctrl-1 = <&spi0_sleep>;
+	pinctrl-names = "default", "sleep";
+};
```

Comment:
why would these be assigned to cpunet out of interest, and not cpuapp?

Reply:
I did this because the network core can also have spi and i2c interfaces. When just implementing on network core, that can be helpful. But the main reason was, that Nordic also did this for network core, when looking into the nrf5340dk_nrf5340_cpunet.dts